service/src/test/java/bio/terra/cbas/controllers/TestRunSetsApiController.java

```diff
@@ -499,6 +505,112 @@ void getRunSetsApiTest() throws Exception {
     assertEquals(
         CbasRunSetStatus.toCbasRunSetApiState(CbasRunSetStatus.RUNNING), runSetDetails2.getState());
   }
+
+  @Test
+  void testRunSetAbort() throws Exception {
```

Comment:
It would be great to add another test case where abort request for 1 run fails and the other one succeeds. We should verify in that test case that the status of that particular run whose abort request failed was not Canceling.